### DISCUSSION TITLE
Changing image resolution.

### DISCUSSION THREAD

**Discussion by veggicide** [raspberrypi/picamera2#656] [Q&A]
Hi, sorry if this is a simple question - I've been trying to figure it but with no luck.  I just bought a camera module 3 and was trying to make my picamera code work with picamera2.

Before when capturing to a BytesIO object I could do this and resize the image.  
compi = BytesIO()
camera.capture(compi, format='jpeg', resize=(320 , 240))

However in picamera2 I just can't figure out how to do this as the resize command doesn't work.  
data = io.BytesIO()
picam2.capture_file(data, format='jpeg')

Any tips on how to resize the images? Other on when I capture the image or when the camera is configured.  I seem to be able to do this with video and the preview part, but not the images themselves.

Thanks!

**Comment by Dun13570** [comment]
Bonjour, avez-vous trouvé la solution s'il vous plaît ?
